src/proposal.mligo

```diff
@@ -402,6 +486,7 @@ let flush(n, config, store : nat * config * storage): return =
 let drop_proposal (proposal_key, config, store : proposal_key * config * storage): return =
   let proposal = check_if_proposal_exist (proposal_key, store) in
   let proposal_is_expired = is_time_reached (proposal, config.proposal_expired_time) in
+  let store = update_quorum(store, config) in
```

Comment:
I think this is no longer necessary?

If I'm not mistaken the quorum threshold is not used here, nor the staked total updated.

Reply:
Removed it.